research/src/pages/popup/popup.proposal.mdx

```diff
@@ -0,0 +1,350 @@
+---
+menu: Proposals
+name: Popup (Editor's Draft)
+path: /components/popup
+pathToResearch: /components/popup.research
+---
+
+import '../../styles/spec.css'
+
+**Editors:**
+
+- Mason Freed (Google)
+- Melanie Richards (Microsoft)
+
+## HTML Incubation Text
+
+_Target venue: [HTML Living Standard](https://html.spec.whatwg.org/multipage/)_
+
+<dl class="element">
+  <dt>
+    <span data-x="concept-element-categories">Categories</span>:
+  </dt>
+  <dd>
+    <span>Flow content</span>.
+  </dd>
+  <dd>
+    <span>Sectioning root</span>.
+  </dd>
+  <dt>
+    <span data-x="concept-element-contexts">Contexts in which this element can be used</span>:
+  </dt>
+  <dd>
+    Where <span>flow content</span> is expected.
+  </dd>
+  <dt>
+    <span data-x="concept-element-content-model">Content model</span>:
+  </dt>
+  <dd>
+    <span>Flow content</span>.
+  </dd>
+  <dt>
+    <span data-x="concept-element-attributes">Content attributes</span>:
+  </dt>
+  <dd>
+    <span>Global attributes</span>
+  </dd>
+  <dd>
+    <code>anchor</code> — Creates an id-reference relationship with another element, which is used
+    for both anchor positioning and popup stack management
+  </dd>
+  <dd>
+    <code>delegatesfocus</code> — Whether to move focus to the popup's first focusable element when
+    the popup is shown
+  </dd>
+  <dd>
+    <code>initiallyopen</code> — Whether the popup is shown upon document load
+  </dd>
+  <dt>
+    <span data-x="concept-element-accessibility-considerations">Accessibility considerations</span>:
+  </dt>
+  <dd>
+    Pending specification in <a href="https://w3c.github.io/html-aria/">ARIA in HTML</a> and{' '}
+    <a href="https://w3c.github.io/html-aam/">HTML-AAM</a>.
+  </dd>
+</dl>
+
+<p class="todo">
+  Add IDL to frontmatter (<a href="https://github.com/openui/open-ui/issues/356">#356</a>)
+</p>
+
+A `popup` element represents transient application UI that “pops up” in the
+[top layer](https://fullscreen.spec.whatwg.org/#top-layer) of a web document, enabling the user to
+perform some task in a timely and immediate fashion. `popup` is similar to the `dialog` element, but
+is distinguished by its transient nature (“light dismiss” behaviors) and mutual exclusivity: only
+one `popup` element may be rendered at a time, with the exception of nested `popup`s.
+
+<p class="note">
+  Examples of popups include: button menus, listboxes used for item selection e.g. comboboxes,
+  "teaching" UI, and other progressive-disclosure widgets or content pickers.
+</p>
+
+## Showing a `popup` element
+
+A `popup` element and its contents should not be rendered unless:
+
+- The `initiallyopen` attribute is specified
```

Comment:
This might be bike-sheddy so feel free to ignore for now, but should it be `initiallyOpen`?

Reply:
Since attributes are case-insensitive, either will work. Camel-casing seems easier to read but looks like the HTML spec has a stylistic preference for all lowercase. Will let HTML editors chime in here!